### PR TITLE
Prepare 1.0.0 release

### PR DESCRIPTION
**Pull request by esune**
Bump versions for 1.0.0 release

### CODE REVIEW

File: charts/traction/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Traction
 
-![version: 0.3.4](https://img.shields.io/badge/Version-0.3.4-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 0.5.1](https://img.shields.io/badge/AppVersion-0.5.1-informational?style=flat-square)
+![version: 0.3.4](https://img.shields.io/badge/Version-0.3.4-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 1.0.0](https://img.shields.io/badge/AppVersion-1.0.0-informational?style=flat-square)
```
Comment (loneil) [inline charts/traction/README.md:3]: Should be version 0.5.1 for the badge?

Comment (loneil) [reply charts/traction/README.md:3]: Or no, that's chart version right?

Comment (esune) [reply charts/traction/README.md:3]: The app version is `1.0.0` - the chart version is `0.3.4` and was bumped after the prior release.

**Review by loneil — APPROVED** [review]

**Comment by esune** [comment]
Something odd is happening with the GHCR authentication,. which is causing the checks to fail.
@i5okie did our token maybe expire or something related to GHCR authentication?

**Comment by esune** [comment]
> Something odd is happening with the GHCR authentication,. which is causing the checks to fail. @i5okie did our token maybe expire or something related to GHCR authentication?

Nevermind, as @loneil just pointed out this is because I opened the PR from my fork... 🤦🏻

**Pull request merged**